src/services/json-api-datastore.service.ts

```diff
@@ -283,7 +283,9 @@ export class JsonApiDatastore {
                     return modelsTypes[property.relationship] === model.constructor;
                 });
                 if (propertyHasMany) {
+                  if (relationshipModel[propertyHasMany.propertyName] != undefined) {
```

Comment:
The build fails because this should be ´!==´

ERROR: src/services/json-api-datastore.service.ts[286, 71]: != should be !==